app/views/proposals/_proposal_form.html.haml

```diff
@@ -55,7 +55,7 @@
     %p.text-right
       = link_to '#description', 'data-toggle' => 'collapse' do
         Do you require something special?
-    .collapse#description
+    #description{class: "collapse #{'in' if @event.description.present?}"}
```

Comment:
For styling purposes, and better readability, please add spaces inside your curly brackets.